proj/hps_accel/src/tensorflow/lite/kernels/internal/reference/integer_ops/conv_accel.cc

```diff
@@ -111,14 +113,25 @@ void ConvPerChannel4x4(const ConvParams& params,
                          MAX_INPUT_WORDS);
         hps_accel::LoadInput(input_width, input_depth, current_input_data);
 
+        // Calculate all outputs for a single output pixel
         for (int out_channel = out_channel_offset;
              out_channel < out_channel_offset + output_channels;
              ++out_channel) {
-          int iterations = filter_height * filter_width * input_depth / 16;
+                         int iterations = filter_height * filter_width * input_depth / 16;
```

Comment:
Slightly weird indentation here but you probably fix this in a subsequent commit so no big deal

Reply:
Yes. I double checked that it is fixed.

Reply:
Actually, it started to bother me, so I fixed it.